assets/prototype/domain/eventEditor/data/types.ts

```diff
@@ -17,15 +17,18 @@ export interface Datetime extends Entity {
 	description?: string;
 	endDate?: string;
 	isActive?: boolean;
+	isDeleted?: boolean; // TODO: refactor according to resolution from here: https://github.com/eventespresso/event-espresso-core/issues/2089
 	isExpired?: boolean;
 	isPrimary?: boolean;
 	isSoldOut?: boolean;
 	isUpcoming?: boolean;
 	length?: number;
 	order?: number;
+	regLimit?: number;
```

Comment:
There is already `capacity` field in GQL which represents that.

Reply:
Refactored: https://github.com/eventespresso/event-espresso-core/pull/2087/commits/e7417d03625da56e3c187543ffff11214e7f6d94